packages/server/src/types/accounts-server-options.ts

```diff
@@ -35,4 +36,5 @@ export interface AccountsServerOptions {
    * Creates a new session token each time a user refreshes his access token
    */
   createNewSessionTokenOnRefresh?: boolean;
+  jwtPayloadCreator?: JwtPayloadCreator;
```

Comment:
I guess this can be just a simple function directly, what do you think?
Also would be great to add a description to the option, for the generated documentation.

Reply:
@pradel  I have changed jwtPayloadCreator to simple function `createPayload`